app/models/organization.py

```diff
@@ -0,0 +1,41 @@
+from .. import db
```

Comment:
can you pull from master and remove these files (model files) from your commit?

Reply:
discussed over slack. for ref: its required for this PR to work.